diagnostic.md

```diff
@@ -21,13 +21,32 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --request GET http://www.example.com/donuts
+
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function getDonuts() {
+    debugger;
+    $.ajax({
+        type: "GET",
+        url: "http://www.example.com/donuts",
+        data: {donuts:list},
```

Comment:
`data` is used for sending data. Here, we're trying to retrieve data, not send it.